src/app/dashboard/commissions/[slug]/[order_id]/kanban.tsx

```
@@ -208,26 +204,26 @@ export function Kanban() {
         if (isActiveTask && isOverTask) {
-            set_tasks((prev) => {
-                const active_index = prev.findIndex((task) => task.id === active_id)
-                const over_index = prev.findIndex((task) => task.id === over_id)
+            setTasks((prev) => {
+                const activeIndex = prev.findIndex((task) => task.id === activeId)
+                const overIndex = prev.findIndex((task) => task.id === overId)
 
-                if (prev[active_index]?.container_id !== prev[over_index]?.container_id) {
-                    prev[active_index]!.container_id = prev[over_index]!.container_id
+                if (prev[activeIndex]?.containerId !== prev[overIndex]?.containerId) {
+                    prev[activeIndex]!.containerId = prev[overIndex]!.containerId
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)